### PR TITLE
Fix bad subtraction with undecorated list

### PR DESCRIPTION
**Pull request by 007lva**
Required for: https://github.com/Soluciones/emergia/pull/23

### CODE REVIEW

File: app/controllers/news_tematica/news_tematicas_controller.rb [hunk 1]
```diff
@@ -103,7 +103,7 @@ def newstematica_klass
     def carga_variables_preview(news_tematica)
       @news_tematica = news_tematica.decorate
       @news_tematica.suscribible_type ||= params[:news_tematica][:suscribible_id].split('-').second
-      todos_los_titulares = @news_tematica.titulares
+      todos_los_titulares = ContenidoEnNewsDecorator.decorate_collection(@news_tematica.titulares)
       @titulares = ContenidoEnNewsDecorator.decorate_collection(todos_los_titulares[0..4])
```
Comment (Fernan2) [inline app/controllers/news_tematica/news_tematicas_controller.rb]: I guess that `ContenidoEnNewsDecorator.decorate_collection` can be removed here...

Comment (007lva) [inline app/controllers/news_tematica/news_tematicas_controller.rb]: @Fernan2 done

**Comment by maxvlc** [comment]
:+1:

**Comment by Fernan2** [comment]
+1

**Pull request merged**